core/src/main/java/de/danoeh/antennapod/core/service/QuickSettingsTileService.java

```diff
@@ -0,0 +1,76 @@
+package de.danoeh.antennapod.core.service;
+
+import android.content.ComponentName;
+import android.content.Intent;
+import android.os.Build;
+import android.os.IBinder;
+import android.service.quicksettings.Tile;
+import android.service.quicksettings.TileService;
+import android.util.Log;
+import android.view.KeyEvent;
+
+import androidx.annotation.RequiresApi;
+
+import de.danoeh.antennapod.core.preferences.PlaybackPreferences;
+import de.danoeh.antennapod.core.receiver.MediaButtonReceiver;
+import de.danoeh.antennapod.core.service.playback.PlaybackService;
+
+@RequiresApi(api = Build.VERSION_CODES.N)
+public class QuickSettingsTileService extends TileService {
+    /**
+     * Logging tag
+     * Missing final 'e' because of max 23 characters for log tags
+     */
+    private static final String TAG = "QuickSettingsTileServic";
```

Comment:
I think we use `Srvc` elsewhere in the app to shorten the tag

Reply:
I found "DownloadSvcNotification" and "LclPlaybackSvcMPlayer" so I'll change it to "QuickSettingsTileSvc"